helm/starboard-app/templates/flow-schema.yaml

```diff
@@ -0,0 +1,165 @@
+        - services
+        - resourcequotas
+        - limitranges
+      verbs:
+        - get
+        - list
+        - watch
+      clusterScope: true
+    - apiGroups:
+        - ""
+      resources:
+        - nodes
+      verbs:
+        - get
+        - list
+        - watch
+      clusterScope: true
+    - apiGroups:
+        - ""
+      resources:
+        - configmaps
+        - secrets
+        - serviceaccounts
+      verbs:
+        - list
+        - watch
+        - get
+        - create
+        - update
+      clusterScope: true
+    - apiGroups:
+        - ""
+      resources:
+        - secrets
+      verbs:
+        - delete
+      clusterScope: true
+    - apiGroups:
+        - ""
+      resources:
+        - events
+      verbs:
+        - create
+      clusterScope: true
+    - apiGroups:
+        - apps
+      resources:
+        - replicasets
+        - statefulsets
+        - daemonsets
+        - deployments
+      verbs:
+        - get
+        - list
+        - watch
+      clusterScope: true
+    - apiGroups:
+        - batch
+      resources:
+        - jobs
+        - cronjobs
+      verbs:
+        - get
+        - list
+        - watch
+      clusterScope: true
+    - apiGroups:
+        - rbac.authorization.k8s.io
+      resources:
+        - roles
+        - rolebindings
+        - clusterroles
+        - clusterrolebindings
+      verbs:
+        - get
+        - list
+        - watch
+      clusterScope: true
+    - apiGroups:
+        - apiextensions.k8s.io
+      resources:
+        - customresourcedefinitions
+      verbs:
+        - get
+        - list
+        - watch
+      clusterScope: true
+    - apiGroups:
+        - batch
+      resources:
+        - jobs
+      verbs:
+        - create
+        - delete
+      clusterScope: true
+    - apiGroups:
+        - networking.k8s.io
+      resources:
+        - networkpolicies
+        - ingresses
+      verbs:
+        - get
+        - list
+        - watch
+      clusterScope: true
+    - apiGroups:
+        - policy
+      resources:
+        - podsecuritypolicies
+      verbs:
+        - get
+        - list
+        - watch
+      clusterScope: true
+    - apiGroups:
+        - aquasecurity.github.io
+      resources:
+        - vulnerabilityreports
+        - configauditreports
+        - clusterconfigauditreports
+        - ciskubebenchreports
+        - clustercompliancereports
+        - clustercompliancedetailreports
+      verbs:
+        - get
+        - list
+        - watch
+        - create
+        - update
+        - delete
+      clusterScope: true
+    - apiGroups:
+        - aquasecurity.github.io
+      resources:
+        - clustercompliancereports/status
+      verbs:
+        - update
+      clusterScope: true
+    subjects:
+    - kind: ServiceAccount
+      serviceAccount:
+        name: {{ index .Values "starboard-app" "serviceAccount" "name" }}
+        namespace: {{ .Release.Namespace }}
+        clusterScope: true
```

Comment:
`clusterScope` is not part of the schema `https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.24/#serviceaccountsubject-v1beta2-flowcontrol-apiserver-k8s-io`